src/AcmTable/AcmTable.tsx

```diff
@@ -172,16 +175,19 @@ export interface AcmTableProps<T> {
     setSearch?: (search: string) => void
     sort?: ISortBy | undefined
     setSort?: (sort: ISortBy | undefined) => void
+    showToolbar?: boolean
     gridBreakPoint?: TableGridBreakpoint
     perPageOptions?: PerPageOptions[]
     autoHidePagination?: boolean
 }
-export function AcmTable<T>(props: AcmTableProps<T>) {
+export function AcmTable<T, ST = void>(props: AcmTableProps<T, ST>) {
     const { items, columns, keyFn, bulkActions = [], rowActions = [], tableActions = [] } = props
-    const sortIndexOffset = bulkActions && bulkActions.length ? 1 : 0
+    const sortIndexOffset = bulkActions && bulkActions.length ? (props.subItems?.length ? 2 : 1) : 0
```

Comment:
This is ugly; but when you add the onCollapse the PF table will reserve a th in the Table; which off sets the index count on the header rows, which ends up messing up the sorting behavior =/